docs/examples/hybrid_solver_service.rst

```diff
@@ -60,7 +62,7 @@ Formulate the Problem
 =====================
 
 For a social graph, `G`, this example simply builds a random sparse graph---using the
-`NetworkX <https://networkx.github.io/>`_ `random_geometric_graph()` function, which
+`NetworkX <https://networkx.github.io/>`_ :code:`random_geometric_graph()` function, which
```

Comment:
why not `:func:`random_geomatric_graph`?

Reply:
Using `:func:`random_geomatric_graph`` results in what looks like a link but is not:
![image](https://user-images.githubusercontent.com/34041130/92528163-263fc000-f1dd-11ea-9b0e-86afafb12a36.png)

I can link it using `:func:`~networkx.generators.geometric.random_geometric_graph()``. I've made the change. It has the disadvantage that if NetworkX restructures, it breaks.

Reply:
I _think_ ```:func:`~networkx.random_geometric_graph` ``` should work

Reply:
Unfortunately not

Reply:
I'd encourage linking with function docs even for 3rd party packages. We mention NetworkX functions below in a few places as well.